editor/src/components/inspector/sections/layout-section/layout-system-subsection/split-chained-number-input.tsx

```diff
@@ -214,6 +222,70 @@ const handleSplitChainedEvent =
       unsetProperty(element, longhand.L),
     ]
 
+    function actionsOrUnset(
+      actions: EditorAction[],
+      unset: boolean,
+      unsetPaths: PropertyPath[],
+    ): EditorAction[] {
+      if (unset) {
+        return unsetPaths.map((path) => unsetProperty(element, path))
+      }
+      return actions
+    }
+
+    function shouldUnset(): boolean {
+      switch (e.type) {
+        case 'one-value':
+          return e.value == null
+        case 'two-value':
+          const otherDirections = useShorthand
+            ? e.value.type === 'V'
+              ? [aggregates.horizontal]
+              : [aggregates.vertical]
+            : e.value.type === 'V'
+            ? [aggregates.top, aggregates.bottom]
```

Comment:
I believe this part of the ternary has been incorrectly flipped. If `e.value.type === 'V'` then the other directions would be `[aggregates.right, aggregates.left]`

Reply:
nope, this is correct - so that it deletes values on the same direction (probably worth adding a comment)

Reply:
In that case, why is the above flipped?
```
e.value.type === 'V'
  ? [aggregates.horizontal]
  : [aggregates.vertical]
```

Reply:
this branch doesn't exist anymore as per https://github.com/concrete-utopia/utopia/pull/3279/commits/5893cd4fcdded5c0e457019b68c9e2ed2324e27f